### PR TITLE
[Ignore] PR into PR #103

### PR DESCRIPTION
**Pull request by wild-endeavor**

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -199,26 +199,26 @@ func newK8sEventRecorder(ctx context.Context, kubeclientset kubernetes.Interface
 func New(ctx context.Context, cfg *config.Config, kubeclientset kubernetes.Interface, flytepropellerClientset clientset.Interface,
 	flyteworkflowInformerFactory informers.SharedInformerFactory, kubeClient executors.Client, scope promutils.Scope) (*Controller, error) {
 
-	var wfLauncher launchplan.Executor
+	var launchPlanActor launchplan.AdminAccess
```
Comment (kumare3) [inline pkg/controller/controller.go:202]: `AdminAccess` ->can we just call if `FlyteAdmin`

File: pkg/controller/nodes/subworkflow/launchplan/admin.go [hunk 2]
```diff
@@ -24,6 +24,8 @@ import (
 
 // Executor for Launchplans that executes on a remote FlyteAdmin service (if configured)
 type adminLaunchPlanExecutor struct {
+	Executor
```
Comment (kumare3) [inline pkg/controller/nodes/subworkflow/launchplan/admin.go:27]: I am confused, isnt this the actual struct?

Comment (kumare3) [reply pkg/controller/nodes/subworkflow/launchplan/admin.go:27]: or is admin client implementing both methods. If so why we do need to compose of the actual interfaces?

Comment (wild-endeavor) [reply pkg/controller/nodes/subworkflow/launchplan/admin.go:27]: I can delete this.

**Comment by kumare3** [comment]
Looks good to me

**Pull request merged**